README.md

```diff
@@ -1,2 +1,71 @@
 # saritasa-s3-tools
-Tools For S3 Used By Saritasa
+
+![GitHub Workflow Status (with event)](https://img.shields.io/github/actions/workflow/status/saritasa-nest/saritasa-s3-tools/checks.yml)
+![PyPI](https://img.shields.io/pypi/v/saritasa-s3-tools)
+![PyPI - Status](https://img.shields.io/pypi/status/saritasa-s3-tools)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/saritasa-s3-tools)
+![PyPI - License](https://img.shields.io/pypi/l/saritasa-s3-tools)
+![PyPI - Downloads](https://img.shields.io/pypi/dm/saritasa-s3-tools)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
+Extension for django and drf to ease work with S3
```

Comment:
Should fastapi be mentioned here?

Reply:
For now nethier django nor fastapi should be mentioned here) Fixed